data/json/monster_special_attacks/spells.json

```diff
@@ -279,5 +279,43 @@
     "field_chance": 2,
     "min_field_intensity": 2,
     "max_field_intensity": 2
+  },
+  {
+    "type": "SPELL",
+    "id": "wraith_attack",
+    "name": { "str": "Wraith Attack" },
+    "description": "Drains some of your essence.",
+    "valid_targets": [ "hostile", "ground" ],
+    "max_level": 1,
+    "flags": [ "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "base_casting_time": 75,
+    "min_range": 1,
+    "max_range": 1,
+    "min_damage": -25,
+    "max_damage": -50,
+    "damage_increment": -10.0,
+    "shape": "blast",
+    "effect": "recover_energy",
+    "effect_str": "FATIGUE"
```

Comment:
FATIGUE (as in sleepiness) or STAMINA?

Reply:
Sleepiness. Was aiming for these to be plentiful but not really that bad in the short term, and to combo with the impossible shapes a bit.  That said this is all still ongoing and I might add some stamina drain as well.